### ISSUE TITLE
Initial fields are duplicated when new row added.

### ISSUE THREAD

**Issue by danpaz** [crucialfelix/django-ajax-selects#94]
This plugin initializes the field using a `data-plugin-options` attribute, which is a stringified object with a property called `initial`. The initial array is used to populate the field when instantiated. The bug arises when any new row is added, which kicks off the same javascript that initializes the regions field.

* Click handler that will trigger when a new row is added: https://github.com/crucialfelix/django-ajax-selects/blob/master/ajax_select/static/ajax_select/js/ajax_select.js#L98-L102.

* Offending stanza that re-appends the initial values every time: https://github.com/crucialfelix/django-ajax-selects/blob/master/ajax_select/static/ajax_select/js/ajax_select.js#L98-L102.

Perhaps after initialization, an attribute could be added to indicate the element should not be initialized again.

**Comment by arthurdejong** [comment]
I'm seeing something very similar to the original bug report so I'm adding it here.

I use an autocompleteselect widget in a tabular formset in the Django admin interface. The field is pre-populated (editing an existing object) so the information in data-plugin-options contains an initial value.

I then select the icon to remove the selection. The hidden input gets correctly updated and the preview is removed.

However, if I click "Add another ..." in the formset the widget is re-initialised and the code pointed out by danpaz is called which updates the results_on_deck div and shows the previous item as selected (granted the item is not really selected as the value of the hidden form element remains empty).

I have worked around this by doing:
```javascript
$('.results_on_deck').bind('killed', function() {
  if ($(this).siblings('input[data-ajax-select=autocompleteselect]').length) {
    var input = $(this).siblings('input[data-ajax-select=autocompleteselect]')
    var opts = JSON.parse(input.attr('data-plugin-options'));
    delete opts['initial'];
    input.attr('data-plugin-options', JSON.stringify(opts));
  }
});
```

I don't use autocompleteselectmultiple so I'm not seeing the same issue as danpaz but the solution should be similar: when re-initialising the widget be more careful to not update results-on-deck.

Thanks!

**Comment by onyekaa** [comment]
Just noticed this issue. I'm using `autocompleteselect` on the frontend. If I'm editing an existing form with an inline formset, for instance 

     1. John Doe   29
     2. Jane Doe   30
     3. Jim Doe     40

If I make any changes to those initial items, then add a new row, all the items are returned to their initial values.

**Comment by jmfederico** [comment]
I think the is a duplicate of #145, which is fixed by #146

**Comment by crucialfelix** [comment]
@jmfederico trigger.reset() was only added in the last release. so this is definitely not related.

**Comment by jmfederico** [comment]
AH! didn't see the date!

**Comment by crucialfelix** [comment]
This was fixed in 1.4.3
